main.js

```diff
@@ -89,3 +94,7 @@ window.onload = function() {
 		element.style.transform = 'rotate(' + deg + 'deg) scale(' + sx + ', ' + sy + ')';
 	}
 };
+
+window.onpopstate = function(event) {
```

Comment:
Wow yeah that is a hack. According to [Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onpopstate) its not even supported on firefox

I like it